### PR TITLE
make latest releases section more robust

### LINKED ISSUE #1936: latest releases doesn't show latest release
Please only report technical bugs. Clarify design issues in the slack channel #ux-ui-support.

## Expected Behavior

It should show the latest released components.
## Current Behavior

Screenshot 2020-08-28 at 11 40 44

## Steps to Reproduce

It is not reproducible any more, because there were several releases after.
1. Got to https://patterns.axa.ch/?path=/story/welcome--to-pattern-library
2. Scroll down
3.
4.

## Possible Solution

Store latest "real" releases and if API gets `Revert "Publish"...` show stored releases

### PR DESCRIPTION
**Pull request by raphaellueckl**
Fixes #1936

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.

### CODE REVIEW

File: src/other/pages/what-is-new/index.scss [hunk 1]
```diff
@@ -7,7 +7,14 @@
...
\ No newline at end of file
+.what-is-new {
+    &__text-with-link {
+        @include text(t2);
+        max-width: 750px;
+    }
+
+    &__response-date {
+        // The important flag is needed, because the component iself sets its styles to the host element internally.
+        margin-bottom: 20px !important;
```
Comment (NoahLiechti) [inline src/other/pages/what-is-new/index.scss:18]: could wrap header in a <div> and set styles there

File: src/other/pages/what-is-new/story.js [hunk 2]
```diff
@@ -73,8 +80,8 @@ export const WhatIsNew = () => {
       </p>
       
       <axa-heading rank="2" variant="secondary">Last releases</axa-heading>
-        <axa-heading rank="6"><span id="githubResponseDate"></axa-heading>
-        <axa-text variant="size-2"><span id="githubResponse"></span></axa-text>
+      <axa-heading rank="6" class="what-is-new__response-date"><span id="githubResponseDate"></axa-heading>
```
Comment (01231) [inline src/other/pages/what-is-new/story.js]: would also rename id's to match BEM notation

File: src/other/pages/what-is-new/index.scss (hunk 1 above)
Comment (01231) [inline src/other/pages/what-is-new/index.scss:18]: could wrap header in a `<div>` and set styles there

**Review by 01231 — CHANGES_REQUESTED** [review]

File: src/other/pages/what-is-new/index.scss (hunk 1 above)
Comment (raphaellueckl) [reply src/other/pages/what-is-new/index.scss:18]: This would be even worse than an important statement on a BEM class.

**Review by 01231 — APPROVED** [review]

**Pull request merged**